GoogleForCreators/web-stories-wp
# Load newly uploaded videos without a delay

## Summary

Tracks newly uploaded video and loads that without a delay.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

Upload a new video by dragging it to the Page and verify that its loaded without a delay in the library after the uploading has finished.

See the expected before and after [here](https://github.com/google/web-stories-wp/pull/4395#issuecomment-689360386).


---



Fixes #1126
